reference/plugins.rst

```diff
@@ -0,0 +1,282 @@
+.. _plugins_reference:
+
+Plugins
+=======
+
+.. warning::
+
+    This is an EXPERIMENTAL feature released in 1.8. Subject to breaking changes.
+
+The Conan plugins are Python functions that are intended to extend the Conan functionalities and let users customize the client behavior at
+determined execution points.
+
+Storage, activation and sharing
+-------------------------------
+
+Plugins are Python files stored under *~/.conan/plugins* folder and their file name should be the same used for activation.
+
+The activation of the plugins is done in the *conan.conf* section named ``[plugins]``. The plugin names listed under this section will be
+considered activated.
+
+.. code-block:: text
+   :caption: *conan.conf*
+
+    ...
+    [plugins]
+    attribute_checker
+    conan-center
+
+They can be easily activated and deactivated from the command line using the :command:`conan config set` command:
+
+.. code-block:: bash
+
+    $ conan config set plugins.attribute_checker  # Activates 'attribute_checker'
+
+    $ conan config rm plugins.attribute_checker  # Deactivates 'attribute_checker'
+
+There is also an environment variable ``CONAN_PLUGINS`` to list the active plugins. Plugins listed in *conan.conf* will be loaded into
```

Comment:
I can't understand what will happen if I have some plugins listed in `conan.conf` and a different set in `CONAN_PLUGINS``.